src/probnum/linalg/solvers/beliefs/_linear_system_belief.py

```diff
@@ -152,3 +153,12 @@ def _induced_x(self) -> randvars.RandomVariable:
         :math:`H` and :math:`b`.
         """
         return self.Ainv @ self.b
+
+    def _induced_Ainv(self) -> randvars.RandomVariable:
+        r"""Induced belief about the inverse from a belief about the solution.
+
+        Computes a consistent belief about the inverse from a belief about the solution.
+        """
+        return randvars.Constant(
+            linops.Scaling(factors=0.0, shape=(self._x.shape[0], self._x.shape[0]))
+        )
```

Comment:
Isn't this a little dangerous, especially given the docstring?
How about:

```suggestion
        raise NotImplementedError()
```

Reply:
I removed the induced Ainv belief since this is not yet needed. Instead the solution-based belief update which iteratively builds up an approximation to the inverse now creates an inconsistent belief with the solution. This is okay for now since we explicitly do not guarantee this. In a future PR we can implement the rank-1 prior mean construction for A and H in Proposition S5 of https://arxiv.org/pdf/2010.09691.pdf from an initial x_0.